src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1485,23 +1537,53 @@ public OFFlowMod buildArpInputCustomerFlow(DatapathId dpid, int port) throws Swi
     }
 
     @Override
-    public List<OFFlowMod> buildExpectedServer42Flows(
-            DatapathId dpid, boolean server42FlowRttFeatureToggle, boolean server42FlowRttSwitchProperty,
+    public List<OFFlowMod> buildExpectedServer42FlowRttFlows(
+            DatapathId dpid, boolean multiTable,
+            boolean server42FlowRttFeatureToggle, boolean server42FlowRttSwitchProperty,
             Integer server42Port, Integer server42Vlan, org.openkilda.model.MacAddress server42MacAddress,
             Set<Integer> customerPorts) throws SwitchNotFoundException {
 
         List<SwitchFlowGenerator> generators = new ArrayList<>();
         if (server42FlowRttFeatureToggle) {
-            generators.add(switchFlowFactory.getServer42TurningFlowGenerator());
+            generators.add(switchFlowFactory.getServer42FlowRttTurningFlowGenerator());
 
             if (server42FlowRttSwitchProperty) {
-                for (Integer port : customerPorts) {
-                    generators.add(switchFlowFactory.getServer42InputFlowGenerator(
-                            server42Port, port, server42MacAddress));
+                if (multiTable) {
```

Comment:
You must remove it because Input rules can be installed in single table mode. See comment above.

Reply:
Ok. I've reverted the changed related to Flow RTT and multitable.